apl/aggregation-function/arg-max.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: arg_max
+description: 'This page explains how to use the arg_max aggregation in APL.'
+---
+
+The `arg_max` aggregation in APL helps you identify the record with the maximum value for a specific numeric field and return one or more additional fields from that record. Use `arg_max` when you want to determine key details associated with a record having the maximum value, such as the longest request duration, highest transaction amount, or most significant span duration.
+
+This aggregation is particularly useful in scenarios like:
+
+- Pinpointing the slowest HTTP requests in log data.
+- Identifying the longest span durations in OpenTelemetry traces.
+- Highlighting the highest severity security alerts in logs.
+
+## For users of other query languages
+
+If you come from other query languages, this section explains how to adjust your existing queries to achieve the same results in APL.
+
+<AccordionGroup>
+<Accordion title="Splunk SPL users">
+
+In Splunk SPL, you use `stats` with a combination of `max` and `by` clauses to achieve similar results. APL provides a dedicated `arg_max` aggregation that simplifies this process.
+
+<CodeGroup>
+```sql Splunk example
+| stats max(req_duration_ms) as max_duration by id, uri
```

Comment:
AFAIK splunk doesn't have an equivalent for arg_max, this is just

| summarize max(req_duration_ms) by id, uri